c7n/resolver.py

```diff
@@ -39,6 +39,10 @@ def __init__(self, session_factory, cache):
         self.cache = cache
 
     def resolve(self, uri):
+        contents = self.cache.get(("uri-resolver", uri))
```

Comment:
I have not done this kind of get before with a tuple

Did you mean?

```
self.cache.get("uri-resolver", uri)
```

Using uri as a fallback?

Reply:
The Pr is correct, the first parameter is a key, in this case a tuple

Reply:
Oh interesting! Thanks for confirming.